client/src/app/components/managing-body/managing-body.component.ts

```diff
@@ -0,0 +1,21 @@
+import {Component, Input, Output, EventEmitter} from '@angular/core';
+
+@Component({
+  selector: 'op-managing-body',
+  templateUrl: 'app/components/managing-body/managing-body.component.html',
+  styleUrls: ['app/components/managing-body/managing-body.component.css'],
+  providers: [],
+  directives: [],
+  pipes: []
+})
+
+export class ManagingBodyComponent {
+  constructor() {}
+
+  @Input() mangingBodyData: Object;
+  @Output() notify : EventEmitter<string> = new EventEmitter();
```

Comment:
@tarieli1 Place the `@Input` and `@Output` at the top of the class [Style 05-14].